jsonschema-validation.xml

```diff
@@ -746,10 +746,23 @@
                     branch of an "allOf" MUST NOT have an impact on a "then"
                     or "else" in another branch.
                 </t>
+                <t>
+                    There is no default behavior for any of these keywords
+                    when they are not present.  In particular, they MUST NOT
+                    be treated as if present with an empty schema, and when
+                    "if" is not present, both "then" and "else" MUST be
+                    entirely ignored.
+                </t>
                 <section title="if">
                     <t>
                         This keyword's value MUST be a valid JSON Schema.
                     </t>
+                    <t>
+                        This validation outcome of this keyword's subschema
+                        has no direct effect on the overall validation
+                        result.  Rather, it controls which of the "then"
+                        or "else" keywords are evaluated.
```

Comment:
/evaluated/are applicable/ ? Thinking we should use applicability termonology where possible.